### DISCUSSION TITLE
Artifactory Hosted Private Variables, Github, and Secrets.

### DISCUSSION THREAD

**Discussion by mikhail-khodorovskiy** [renovatebot/renovate#12890] [Request Help]
We currently use Github app for Renovate and Artifactory for private dependency managements as well as presets for org-wide setup.  In order to avoid committing secrets to the reset repository for the artifactory access.  Is there any way we can use Github org secrets for artifactory access?  What other ways do you recommend to setup the hostRules in order to keep the artifactory credentials private?

**Comment by rarkins** [comment]
Are you self-hosting or using the hosted Renovate app?

**Comment by mikhail-khodorovskiy** [comment]
Renovate App for Cloud Github.

**Comment by rarkins** [comment]
See https://docs.renovatebot.com/getting-started/private-packages/#whitesource-renovate-hosted-app-encryption

Renovate does not request access to GitHub Actions secrets because:
1. You can't request selective access, so you'd need to grant Renovate access to all your secrets
2. Even if you wanted to do that, Renovate can't request selective *permissions* either. So anybody who didn't want to grant access to all secrets would not be able to install the app
